BUILD

```diff
@@ -0,0 +1,6 @@
+load("@rules_cc//cc:defs.bzl", "cc_binary")
+
+cc_binary(
+    name = "hello-world",
+    srcs = ["hello-world.cc"],
+)
```

Comment:
Should we have a `cc_test` too, that could be checked automatically by Travis as a PR check?

Reply:
+1